utils/maps.go

```diff
@@ -0,0 +1,40 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package utils
```

Comment:
Same as with `bytes.go` I feel like we should make sure this is our best option.

Reply:
If we migrated away from specifying flags using the golang-provided `flag.FlagSet` and instead used the richer `github.com/spf13/pflag`, we may be able to avoid some of this casting. Otherwise, it is a function of the CLI and config providing information in different formats. These functions do their best to abstract this complexity.

Happy to take another pass here if you know some command line tricks.